<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>packageSourceDialog</class>
 <widget class="QDialog" name="packageSourceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>452</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>482</width>
    <height>452</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Build options</string>
  </property>
  <widget class="QGroupBox" name="groupBox_1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>140</y>
     <width>474</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Local resources</string>
   </property>
   <widget class="QListWidget" name="listWidget_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>356</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton_1">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>55</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>474</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Remote resources</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>356</width>
      <height>32</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>20</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>55</y>
      <width>356</width>
      <height>66</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QPushButton" name="removeButton_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>55</y>
      <width>96</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="buildButton">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>410</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Build</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>80</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>285</y>
     <width>471</width>
     <height>110</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0" rowminimumheight="0,0,0" columnminimumwidth="0,0,0">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Workspace</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="wsEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Project name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="wsButton">
      <property name="text">
       <string notr="true">...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Pardus installation media</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="pimEdit">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="pimButton">
      <property name="text">
       <string notr="true">...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="pnEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
